proposals/lip-0002.md

```diff
@@ -24,10 +24,10 @@ As the Lisk protocol evolves and gains popularity, a clear increase in usage and
 
 ## Rationale
 
-We propose to set a maximum block size of 15KB (kilobytes). This size will maintain the yearly blockchain growth below 50GB, even when blocks are always at full capacity, and the active delegates do not miss any blocks. The summary of the numerical implications are as follows:
+We propose to set a maximum block size of 15 KiB (15 * 1024 bytes). This size will maintain the yearly blockchain growth below 50 GiB, even when blocks are always at full capacity, and the active delegates do not miss any blocks. The summary of the numerical implications are as follows:
 
-* Maximum block size: 15KB.
-* Blockchain size will grow a **maximum** of 47.3GB per year.
+* Maximum block size: 15 KiB.
+* Blockchain size will grow a **maximum** of 47.3 GiB per year.
```

Comment:
If I'm not wrong, it must be 45.1 if the unit is GiB, as 1 GiB = 1024^3 bytes.
And we have a maximum of `(6*60*24*365)*15*1024 bytes = 48439296000 bytes` per year.

BTW: Do we need to incorporate leap years? :)

Reply:
I just changed the unit name and was assuming the specifications consistently used 1024 as base. I updated this line.